bedrock/products/templates/products/vpn/resource-center/base-article.html

```diff
@@ -0,0 +1,87 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from 'macros-protocol.html' import split %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('vpn-resource-center')}}
+{% endblock page_css %}
+
+{% block page_title%}{% block article_title %}{% endblock article_title %}{% endblock page_title%}
+
+{% block content %}
+<nav class="mzp-c-breadcrumb">
+  <ol class="mzp-c-breadcrumb-list">
+    <li class="mzp-c-breadcrumb-item">
+      <a href="{{ url('products.vpn.resource-center.landing') }}">{{ ftl('vpn-resource-center-title') }}</a>
+    </li>
+    <li class="mzp-c-breadcrumb-item">
+      {{ self.article_title() }}
+    </li>
+  </ol>
+</nav>
+<div class="vpn-article-container">
+  <div class="mzp-l-content mzp-t-content-md">
+      <article class="mzp-c-article vpn-article">
+        {# Categories will be used at some point but not yet #}
+        {# <p class="vpn-article-category">{% block article_category %}{{ self.article_category().capitalize() }}{% endblock %}</p> #}
+        <h2>{{ self.article_title() }}</h2>
+        {% block vpn_article %}{% endblock vpn_article %}
+      </article>
+    <div class="vpn-c-vote no-js">
+      <h3>{{ ftl('vpn-resource-center-was-this-article') }}</h3>
+      <div class="vpn-c-vote-btn-container">
+        <button type="button" data-cta-type="button" data-cta-text="{% block article_slug %}{% endblock %}" class="vpn-c-vote-btn up">
+          <svg width="42" height="38" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+            <path d="M36.54 13.104h2.184A3.277 3.277 0 0 1 42 16.38v17.473a3.277 3.277 0 0 1-3.276 3.276H36.54a3.277 3.277 0 0 1-3.276-3.276V16.38a3.277 3.277 0 0 1 3.276-3.276ZM5.38 10.752c-3.669 0-6.261 3.595-5.1 7.076l4.48 13.44a5.375 5.375 0 0 0 5.1 3.677h19.036V15.238L24.19 8.063l-2.264-6.796A1.857 1.857 0 0 0 20.166 0h-.007a4.032 4.032 0 0 0-4.032 4.032v5.914l-.806.806H5.38Z" fill="currentColor"/>
+          </svg>
+        </button>
+        <button type="button" data-cta-type="button" data-cta-text="{{ self.article_slug() }}" class="vpn-c-vote-btn down">
```

Comment:
```suggestion
        <button type="button" data-cta-type="button" data-cta-text="{{ self.article_slug() }}" class="vpn-c-vote-btn down">
        <span class="visually-hidden">{{ ftl('vpn-resource-center-article-not-helpful') }}</span>
```

Reply:
I think this is only available using a Sass mixin, so you might need to add some CSS to make it work instead of using a class name in HTML:

```css
.vpn-c-vote-btn span {
    @include visually-hidden;
}
```